src/components/Inputs/textField/TextField.tsx

```diff
@@ -13,9 +14,11 @@ interface ITextFieldProps<T extends TextFieldType = "text"> {
     className?: string
     type?: T
     label?: string
+    error?: ValidationErrorItem
+    requiredMessage?: string
```

Comment:
Če prikažeš required message glede na error, zakaj si dal še en dodaten property not? To lahko veš iz error propertyja že. Reši ta problem samo znotraj TextFielda.